### DISCUSSION TITLE
Internationalization of tags

### DISCUSSION THREAD

**Discussion by obelix05** [nextcloud/recognize#197] [Q&A] [answered]
Hi all,
I'm looking for a way to change the tag names created by "Recognize". Is there something I can use for that? The only way I think off was to rename all the labels in the rules.yaml, but I think this is not the best solution.
Any ideas?

Thanks in advance.

**Comment by marcelklehr** [comment, marked as answer]
Hi @obelix05 
This is a pending issue: see #3
